src/containers/AdminCampaignStats.jsx

```diff
@@ -181,6 +229,32 @@ class AdminCampaignStats extends React.Component {
     );
   }
 
+  renderTexterStats() {
+    const data = (this.props.data.campaign.assignments || []).map(summary => ({
+      texter: `${summary.texter.firstName} ${summary.texter.lastName}`,
+      messagedCount:
+        summary.texter.assignment.contactsCount -
+        summary.texter.assignment.needsMessageCount,
+      unmessagedCount: summary.texter.assignment.needsMessageCount,
+      percentComplete: `${Math.round(
+        ((summary.texter.assignment.contactsCount -
+          summary.texter.assignment.needsMessageCount) *
+          100) /
+          summary.texter.assignment.contactsCount
+      )}%`,
+      contactsCount: summary.texter.assignment.contactsCount
+    }));
+    return (
+      <DataTables
```

Comment:
It would be great if all these columns were sortable. Would that be possible?

Reply:
I'd prefer not to in this PR, the DataTables will need a sort function and we'll need to keep the state of the table